build-system/compile/compile.js

```diff
@@ -359,12 +352,6 @@ function compile(
           )
         )
         .on('error', reject)
-        .pipe(
-          gulpIf(
-            shouldAppendSourcemappingURLText,
-            gap.appendText(`\n//# sourceMappingURL=${outputFilename}.map`)
```

Comment:
With this, we can uninstall `gulp-append-prepend` from `package.json`

Reply:
Done.